srv/srv0start.cc

```diff
@@ -612,13 +606,17 @@ static db_err open_or_create_log_file(
     ret = os_file_get_size(files[i], &size);
     ut_a(ret);
 
-    if (size != (off_t) srv_log_file_size) {
+    off_t size_high = srv_calc_high32(srv_log_file_size);
+    off_t file_size = (off_t)srv_calc_low32(srv_log_file_size) + (((off_t)size_high) << 32);
+    if (size != file_size) {
```

Comment:
@sunbains Does this look right? Without this we are comparing 2048 with 32 mb.

Reply:
I think you are right, it's converted to  number of pages from number of bytes in  "db_err srv_normalize_init_values()", better to compare it with srv_log_file_curr_size then which is in bytes.

Reply:
Makes sense! Fixed now :)